src/base/release_string.c

```diff
@@ -100,7 +100,7 @@ static REL_COMPATIBILITY rel_get_compatible_internal (const char *base_rel_str,
 /*
  * Disk (database image) Version Compatibility
  */
-static float disk_compatibility_level = 10.02f;
+static float disk_compatibility_level = 10.03f;
```

Comment:
@lucianlazar2211 

I think this is the best chance to also remove obsoleted stuffs. 

Please remove `PAGE_XASL` from `PAGE_TYPE`. 

I'd like to also change `FILE_TYPE` as follows:
```
typedef enum
{
  FILE_TRACKER,
  FILE_HEAP,
  FILE_HEAP_REUSE_SLOTS,
  FILE_MULTIPAGE_OBJECT_HEAP,
  FILE_BTREE,
  FILE_BTREE_OVERFLOW_KEY,
  FILE_EXTENDIBLE_HASH,
  FILE_EXTENDIBLE_HASH_DIRECTORY,
  FILE_LONGDATA,
  FILE_CATALOG,
  FILE_DROPPED_FILES,
  FILE_VACUUM_DATA,
  FILE_QUERY_AREA,
  FILE_TEMP,
  FILE_UNKNOWN_TYPE,
  FILE_LAST = FILE_UNKNOWN_TYPE
} FILE_TYPE;
```